packages/bsky/src/api/app/bsky/purchase/getFeatures.ts

```diff
@@ -0,0 +1,50 @@
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+import { Features } from '@atproto/api/dist/client/types/app/bsky/purchase/getFeatures'
+
+export default function (server: Server, ctx: AppContext) {
+  server.app.bsky.purchase.getFeatures({
+    auth: ctx.authVerifier.standard,
+    handler: async ({ auth }) => {
+      const viewer = auth.credentials.iss
+
+      const features = await getFeaturesForViewerEntitlements(viewer, ctx)
+
+      return {
+        encoding: 'application/json',
+        body: {
+          features,
+        },
+      }
+    },
+  })
+}
+
+const defaultFeatures: Features = {
+  customProfileColor: false,
+}
+
+const coreEntitlementFeatures: Features = {
+  customProfileColor: true,
+}
+
+const getFeaturesForViewerEntitlements = async (
+  viewerDid: string,
```

Comment:
nit, but i usually do the `ctx` as the first param of functions like this since you can kinda think of it as "parameterizing" the function